src/Vehicle/VehicleFact.json

```diff
@@ -59,6 +59,20 @@
     "decimalPlaces":    1,
     "units":            "m/s"
 },
+{
```

Comment:
VehicleFact.json is used by telemetry values which are exposed though the Vehicle object. This pull is for mission item changes which has no affect on Vehicle telemetry. So this change is incorrect and not needed.